tests/cli/test_cli_add.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+from copy import deepcopy
+import datetime
+
+from pathlib import Path
+import ruamel.yaml
+
+from housekeeper.store import Store
+
+
+def test_cli_add_file_to_bundle(store, store_url, tmpdir, invoke_cli, bundle_data, bundle_file_1, bundle_file_2):
```

Comment:
I suggest having one test per GIVEN/WHEN/THEN to improve readability/maintainability and bug-tracking-time, what do you say?

Reply:
That might be convenient yes